torch/distributed/elastic/multiprocessing/api.py

```diff
@@ -306,28 +311,35 @@ def pids(self) -> Dict[int, int]:
         raise NotImplementedError()
 
     @abc.abstractmethod
-    def _close(self, death_sig: signal.Signals, timeout: int = 30) -> None:
+    def _close(self, death_sig: signal.Signals) -> None:
         r"""
         Terminates all processes managed by this context and cleans up any
         meta resources (e.g. redirect, error_file files).
         """
         raise NotImplementedError()
 
     def close(
-        self, death_sig: Optional[signal.Signals] = None, timeout: int = 30
+        self,
+        death_sig: Optional[signal.Signals] = None,
+        timeout: Optional[float] = None,
     ) -> None:
         r"""
         Terminates all processes managed by this context and cleans up any
         meta resources (e.g. redirect, error_file files).
 
         Args:
             death_sig: Death signal to terminate porcesses.
-            timeout: Time to wait for processes to finish, if process is
+            timeout: DEPRECATED Time to wait for processes to finish, if process is
```

Comment:
since `timeout` has been removed from the method signature, we can also remove the docstring here.